<launch>

    <node pkg="mediapipe_track" type="locker_human.py" name="locker_human" output="screen"/>

    <node pkg="tf" type="static_transform_publisher" name="target_broadcaster" args="0 0 0 0 0 0 camera_link target 15" />

</launch>
